R/metadata.R

```diff
@@ -35,12 +36,14 @@
 #'   library(magrittr)
 #'
 #'   adlb %>%
-#'     xportr_domain_name("adlb") %>%
 #'     xportr_metadata(metadata, "test") %>%
 #'     xportr_type() %>%
 #'     xportr_order()
 #' }
-xportr_metadata <- function(.df, metadata, domain = NULL) {
+xportr_metadata <- function(.df, metadata = NULL, domain = NULL) {
+  if (is.null(metadata) && is.null(domain)) {
+    stop("Must provide either metadata or domain argument")
```

Comment:
This got flagged by codecov as not being covered by a new test. I actually think calling it without metadata or domain is fine, it just wouldn't do anything

Reply:
I think we can figure out the testing later. Unless it can be done quickly?  

Just want to get this merged in so others work can be moved onto